CHANGELOG.md

```diff
@@ -7,14 +7,26 @@ Mapbox welcomes participation and contributions from everyone.
 * Enable support for incremental annotation processing. ([#1323](https://github.com/mapbox/mapbox-maps-android/pull/1323))
 * Make use of non-deprecated common Mapbox logger. ([#1327](https://github.com/mapbox/mapbox-maps-android/pull/1327))
 * Expose factory methods for high-level camera animators. ([#1338](https://github.com/mapbox/mapbox-maps-android/pull/1338))
+* Introduce experimental 3D model support. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
+* Add support for sending turnstile event in native for embedded platforms. ([#1351](https://github.com/mapbox/mapbox-maps-android/pull/1351))
```

Comment:
is this supposed to be a public change log? especially it's for embedded platforms?

Reply:
interesting, checked again and it's not there anymore, and in general a lot of things are re-phrased. Will make use of new ones.